arm_gimbal:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 150  

  # Position Controllers ---------------------------------------
  pitch_position_controller:
    type: effort_controllers/JointPositionController
    joint: p_pitch
    pid: {p: 1, i: 5, d: 0.35}
  roll_position_controller:
    type: effort_controllers/JointPositionController
    joint: p_roll
    pid: {p: 1, i: 1, d: 0.1}

 
